coreos-base/oem-packet/files/units/packet-phone-home.service

```diff
@@ -6,8 +6,9 @@ After=coreos-metadata.service
 
 [Service]
 EnvironmentFile=/run/metadata/coreos
-Type=oneshot
```

Comment:
Not in the OS, and I wouldn't expect in user configs either.  It's a leaf service.